src/App.js

```diff
@@ -16,7 +16,7 @@ import Modal from 'components/Modal'
 import NoMeilisearchRunning from 'components/NoMeilisearchRunning'
 import ApiKeyAwarenessBanner from 'components/ApiKeyAwarenessBanner'
 import getIndexesListWithStats from 'utils/getIndexesListWithStats'
-import shouldDisplayCloudBanner from 'utils/shouldDisplayCloudBanner'
+import isBannerEnabled from 'utils/isBannerEnabled'
```

Comment:
Can we rename it to have `cloud` in its name? 
If tomorrow we want to display an other banner too, it could be confusing :)

Reply:
Thanks for the feedback!